app/Http/Middleware/TrustProxies.php

```diff
@@ -8,23 +8,16 @@
 class TrustProxies extends Middleware
 {
     /**
-     * The trusted proxies for this application,
-     * sourced from 'config/trustedproxy.php'.
+     * The trusted proxies for this application.
      *
      * @var array
      */
     protected $proxies;
 
     /**
-     * The current proxy header mappings.
+     * The headers that should be used to detect proxies.
      *
-     * @var array
+     * @var int
      */
-    protected $headers = [
-        Request::HEADER_FORWARDED => null, // Not set on AWS or Heroku.
-        Request::HEADER_X_FORWARDED_FOR => 'X_FORWARDED_FOR',
-        Request::HEADER_X_FORWARDED_HOST => null, // Not set on AWS or Heroku.
-        Request::HEADER_X_FORWARDED_PORT => 'X_FORWARDED_PORT',
-        Request::HEADER_X_FORWARDED_PROTO => 'X_FORWARDED_PROTO',
-    ];
+    protected $headers = Request::HEADER_X_FORWARDED_ALL;
```

Comment:
I'm less familiar with Trusted Proxy and want to make sure the new bit property is fine that should encompass the prior series of headers.